docs/track.rst

```diff
@@ -2898,6 +2898,48 @@ Meta-data
 
 This operation returns no meta-data.
 
+sql
+~~~~~~~~~~~~~~~~~~~
+
+Executes an SQL query and optionally paginates through subsequent pages. The ``body`` property will directly be passed to
+the `SQL search API <https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-search-api.html>`_.
+
+Properties
+""""""""""
+
+* ``body``: The request body passed to the _sql API with the initial request. Must contain the "query" key. Subsequent pagination requests will only include the cursor in the request body.
```

Comment:
```suggestion
* ``body`` (mandatory): The request body passed to the _sql API with the initial request. Must contain the "query" key. Subsequent pagination requests will only include the cursor in the request body.
```

Reply:
Let's mark this as mandatory. We do that throughout these docs for required params.